jscomp/stdlib-402/hashtbl.ml

```diff
@@ -114,7 +114,7 @@ let resize indexfun h =
 
 let key_index h key =
   (* compatibility with old hash tables *)
-  if Obj.size (Obj.repr h) >= 3
+  if (* XXX *) true || Obj.size (Obj.repr h) >= 3
   then (seeded_hash_param 10 100 h.seed key) land (Array.length h.data - 1)
```

Comment:
we may remove such thing, since we don't need maintain compatibility with old hashtbl

Reply:
Happy to remove.
Does it mean I should remove the false branch and any dead code it could result from that?